samples/compression/lz4/sample.yaml

```diff
@@ -0,0 +1,10 @@
+sample:
+  name: lz4 sample
+  description: lz4 data compression library
+
+tests:
+  sample.compression.lz4:
+    platform_allow: nrf52840dk_nrf52840
+    platform_exclude: native_posix qemu_x86
```

Comment:
I think that either `platform_allow` or `platform_exclude` should be used, not both. BTW tested this sample on `qemu_x86` and everything works great, so no reason to exclude it. It also works on `native_posix`. So both `platform_allow` and `platform_exclude` should be dropped.

Reply:
@mniestroj buildkite failure is caused due to qemu platform's. Shall I include ````platform_allow```` here?

Reply:
QEMU should work. Maybe we miss something like (based on samples/hello_world):
```
    harness: console
    harness_config:
      type: one_line
      regex:
        - "Validation done. (.*)"
```

But then, please make sure that this string that we wait for is printed only in case of success.